packages/rancher-gatekeeper/generated-changes/patch/values.yaml.patch

```diff
@@ -1,35 +1,39 @@
 --- charts-original/values.yaml
 +++ charts/values.yaml
-@@ -1,8 +1,7 @@
+@@ -1,5 +1,5 @@
  replicas: 3
 -auditInterval: 60
 +auditInterval: 300
+ auditMatchKindOnly: false
  constraintViolationsLimit: 20
  auditFromCache: false
--createNamespace: true
- disableValidatingWebhook: false
- validatingWebhookTimeoutSeconds: 3
- enableDeleteOperations: false
-@@ -11,8 +10,8 @@
- emitAdmissionEvents: false
- emitAuditEvents: false
+@@ -16,13 +16,13 @@
+   labelNamespace:
+     enabled: true
+     image:
+-      repository: line/kubectl-kustomize
+-      tag: 1.20.4-4.0.5
++      repository: rancher/kubectl
++      tag: v1.20.2
```

Comment:
This is a net-new change from last one: https://github.com/rancher/charts/blob/dev-v2.5/packages/rancher-gatekeeper/generated-changes/patch/values.yaml.patch
Are we sure that kubectl-kustomize doesn't use kustomize ? Because rancher/kubectl doesn't use it.

Reply:
Yes, this is all it's used for: https://github.com/open-policy-agent/gatekeeper/blob/c8621550a3cc38d11f08b33a5c2b74a9ad328b1f/charts/gatekeeper/templates/namespace-post-install.yaml#L35-L41